pages/linux/grub-mkrelpath.md

```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a file system path relative to the root of its file system.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the file system path relative to the root which can be used by other GRUB utilities:
+
+`grub-mkrelpath {{path/to/relativepath}}`
```

Comment:
this should be a directory, right?
```suggestion
`grub-mkrelpath {{path/to/directory}}`
```

Reply:
I think we should leave the relative path as it is. 

Ref:

```text
The program grub-mkrelpath makes a file system path relative to the root of its containing file system. For instance, if /usr is a mount point, then:

$ grub-mkrelpath /usr/share/grub/unicode.pf2
‘/share/grub/unicode.pf2’
```

Reply:
It's ambigous, it's a `path/to/file_or_directory` then?

Reply:
> It's ambigous, it's a `path/to/file_or_directory` then?

It makes sense and clear for others. Shall we go with this change @kbdharun and @navarroaxel ?

Reply:
> It's ambigous, it's a `path/to/file_or_directory` then?

I am not sure should we ask @CleanMachine1 ?

Reply:
> It's ambigous, it's a `path/to/file_or_directory` then?

@navarroaxel, I have found the following link where the path is mentioned as file name. https://manpages.ubuntu.com/manpages/jammy/man1/grub-mkrelpath.1.html

Should we make it as path_to_file?